src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -17,6 +17,14 @@ const StyledButton = styled(Button)`
     margin: ${size.large} auto;
 `;
 
+const StyledCheckbox = styled(Checkbox)`
+    /* Leafygreen's Checkbox input is given height/width of 0, hiding invalid popup */
```

Comment:
🤷‍♂️  looks like this element was actually hidden with height/width 0. Chrome's error message dialog likely could not bind to it. Still functional in other browsers with this change.

Reply:
Does this still show the custom checkbox? They probably did this to use the events on the checkbox, but to display a totally custom implementation.

Reply:
Yes, this still does!

Reply:
The `input` sits behind the custom element, so the behavior still works as intended